### PR TITLE
Update links and resource contexts version

### PR DESCRIPTION
**Pull request by hygt**
* Fix cyclic reference in resource/standards context

* Update workbench version

* Enable scalafmt check during `review`

### CODE REVIEW

**Comment by bogdanromanx** [comment]
Jenkins retest this please

File: modules/service/src/main/scala/ch/epfl/bluebrain/nexus/kg/service/routes/ContextRoutes.scala [hunk 1]
```diff
@@ -222,6 +222,8 @@ object ContextRoutes {
 class ContextCustomEncoders(base: Uri, prefixes: PrefixUris)(implicit E: Context => ContextId)
     extends RoutesEncoder[ContextId, ContextRef, Context](base, prefixes) {
 
+  private val excluded = List(prefixes.CoreContext.context, prefixes.StandardsContext.context)
```
Comment (MFSY) [inline modules/service/src/main/scala/ch/epfl/bluebrain/nexus/kg/service/routes/ContextRoutes.scala]: I would suggest to only exclude the nexus/core/resource context.

**Comment by bogdanromanx** [comment]
Jenkins retest this please

**Comment by hygt** [comment]
Jenkins retest this please

**Comment by umbreak** [comment]
Jenkins retest this please

**Pull request merged**